src/main/java/frc/robot/utils/MoPrefs.java

```diff
@@ -13,12 +14,29 @@
 
 public final class MoPrefs {
     public static Pref<Double> maxDriveRpm = doublePref("Drive Max RPM", 6000.0);
+    public static Pref<Double> maxDriveSpeed = doublePref("Drive Max Speed m/s", 6000.0);
```

Comment:
I think these two preferences are mutually exclusive. That is, since we have maxDriveSpeed and driveMtrScale, we don't need maxDriveRpm.

I think maxDriveRpm was what we were using for the old mecanum drive pid, but we don't need it anymore for the swerve pid.